sdk/scheduler/build.gradle

```diff
@@ -147,7 +147,7 @@ dependencies {
     compile "javax.el:javax.el-api:${elVer}"
     compile "org.glassfish.web:javax.el:${elVer}"
     compile 'com.mesosphere:mesos-http-adapter:0.4.1'
-    compile 'org.apache.mesos:mesos:1.5.0' // This is the Mesos version used in DC/OS 1.11.
+    compile 'org.apache.mesos:mesos:1.7.1-rc1' // This is the Mesos version used in DC/OS 1.12.1.
```

Comment:
AFAIK this will continue to work with 1.10+. The only incompatible changes are the `CREATE_DISK` and `DESTROY_DISK` operations, which are experimental and not used by SDK at all (The only consumer now is DSS).

Reply:
BTW since Mesos 1.7.1 has been released, I could update this PR to use the official release instead of the release candidate.